packages/ui/src/message-pane-input/MessagePaneInput.jsx

```diff
@@ -21,6 +21,7 @@ import "!style-loader!css-loader!@draft-js-plugins/mention/lib/plugin.css";
 // import suggestionStyles from "./suggestions.module.css"
 import "./message-editor-input.css";
 import Toolbar from "./components/Toolbar";
+import Toolbar2 from "./components/Toolbar2";
 import mentions from "./mentions.data";
```

Comment:
I'd suggest naming the toolbars like this `ToolBarTop` and `ToolBarBottom` so that its easier for a future developer to be able to distinguish them

Reply:
Okay, I'll do that right away

Reply:
I have made the changes, sorry its late.